ewapt.py

```diff
@@ -949,6 +952,10 @@ def check(str):
 				user_data.visiting = target_data.id_user
 				user_data.rr_challenger = ""
 				user_data.persist()
+				client = ewutils.get_client()
+				server = ewcfg.server_list[user_data.id_server]
+				member_object = server.get_member(user_id=user_data.id_user)
```

Comment:
isn't this the same as the message author?

Reply:
dm commands do not have some attributes of a typical message author. this is a way to to thwart that.